instrumentor/src/index.js

```diff
@@ -295,6 +300,43 @@ const instrumentStartup = path => {
   }
 };
 
+const TOUCHABLE_COMPONENTS = [
+  'TouchableOpacity',
+  'TouchableNativeFeedback',
+  'TouchableWithoutFeedback',
+  'TouchableHighlight',
+];
+
+const instrumentTouchableHoc = path => {
+  if (!TOUCHABLE_COMPONENTS.includes(path.node.id.name)) {
+    return;
+  }
+
+  // 'path.node' represents a class *declaration*, so we need to convert 'path.node' to a class *expression* before we can pass it as an
```

Comment:
What is the difference between `declaration` and `expression` here? (An example would be nice! 🙂)

Reply:
It's kinda difficult to explain without looking at the AST the code parses into, since a `ClassDeclaration` and `ClassExpression` _look exactly the same in code_.

I _think_ that a declaration declares the name of the class as an in-scope symbol, whereas a class expression is just a class with a name (the class doesn't actually get declared as an in-scope thing).